src/ubuntu/18.04/helix/sqlserver/amd64/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-amd64
+
+USER root
+
+# Install SqlServer 2019 and FTS
+RUN apt-get update && \
+    apt-get install -qq -y \
+        software-properties-common \
+        wget \
+    && wget -qO- https://packages.microsoft.com/keys/microsoft.asc | apt-key add - && \
+    add-apt-repository "$(wget -qO- https://packages.microsoft.com/config/ubuntu/18.04/mssql-server-2019.list)" && \
+    apt-get update && \
+    apt-get install -qq -y \
+        mssql-server \
+        mssql-server-fts \
+    && rm -rf /var/lib/apt/lists/*
+
+CMD [ "/opt/mssql/bin/sqlservr" ]
```

Comment:
Is your intention really to use `CMD` or would `ENTRYPOINT` be more appropriate?  See https://docs.docker.com/engine/reference/builder/#understand-how-cmd-and-entrypoint-interact for the difference.

Reply:
Either could work. This just starts SqlServer service.
I followed CMD from here https://github.com/microsoft/mssql-docker/blob/42c6ed8fa6d2a1696cf864e91e2660aeb7986dd0/linux/mssql-server-linux/Dockerfile#L15
Can change it to ENTRYPOINT too as this command should be run always for this particular image.

Reply:
It surprise me they do it that way.  But if that's how the official SQL Server image does it, might as well follow that pattern.